### PR TITLE
Speed Rate Button Changes

### PR DESCRIPTION
**Pull request by mlammon11**
Reason: 
- The original speed rate button had small increase/decrease input buttons that would be difficult to use for the visually impaired.

Changes:
- Using whole numbers instead of 1/2 numbers since there didn't seem to be much of a difference by .5
- Incorporated larger increase/decrease buttons to be easier to click
- Kept styling similar to the other buttons to retain feel of the website

### CODE REVIEW

File: app/js/audio/sound.js [hunk 1]
```diff
@@ -94,12 +94,22 @@ export function createSpatialPlayer(locationProvider) {
...
+    setRate(rateChange) {
+      // rateChange will be 1 when increaseRate button is clicked, otherwise will be -1 when decreaseRate button is clicked
+
+      // this allows: 1 <= player.rate <= 5
+      if(rateChange == 1 && player.rate != 5){
+        player.rate += rateChange;
+      }
+      else if (rateChange == -1 && player.rate != 1){
+        player.rate += rateChange;
+      }
```
Comment (steinbro) [inline app/js/audio/sound.js:110]: This would be more straightforward as two separate methods without arguments, increaseRate and decreaseRate.

**Review by steinbro — COMMENTED** [review]
I like how it looks! I've requested some changes in the comments.

**Comment by steinbro** [comment]
Thanks for addressing the feedback! I had to add some logic to get the new widget functional on the GPX simulator, but it's all working now.

**Comment by ragar3** [comment]
Ah, thanks! Was planning to work on that today.

**Pull request merged**